aiobotocore/endpoint.py

```diff
@@ -51,43 +69,43 @@ def __dir__(self):
         return attrs
 
     def __del__(self):
-        self.close()
+        # callers may not read the Body resulting in an unclosed ClientResponse
+        if self._loop.is_running():
+            asyncio.ensure_future(self.__response.release(), loop=self._loop)
```

Comment:
actually ya that's a better example, also I found an issue with the new properties as it seems someone is trying to set them, working on fix